src/packages/auth/src/authentication/token/auth-token.ts

```diff
@@ -110,9 +110,8 @@ export class AuthTokenProvider implements BaseAuthTokenProvider {
 	}
 }
 
-export const verifyAndCreateTokenFromAuthToken = (authToken: AuthToken) => {
+export const verifyToken = (authToken?: AuthToken) => {
```

Comment:
Should we remove this function now as its only checking if undefined. That check could happen in the `stepUpToken` and `generateToken` functions on `tokenProvider`

Reply:
Done in a9ce9b9b3a5235265e10d18fcbcb2c2dd9119f6f